listen_unix_setopt.go

```diff
@@ -0,0 +1,7 @@
+//go:build linux || openbsd || darwin
```

Comment:
As we're unraveling the `unix` build tag value, we probably need to consider all of these individually now :(

https://go-review.googlesource.com/c/go/+/389934/7/src/go/build/syslist.go

Unless there's a way to say "unix but not freebsd"